.github/workflows/examples-telink.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: connectedhomeip/chip-build-telink:0.5.59
+            image: connectedhomeip/chip-build-telink:0.5.74
```

Comment:
This must be done in the next PR (after building the docker)

Reply:
@Damian-Nordic Fixed